router/core/router.go

```diff
@@ -870,7 +875,7 @@ func (r *Router) newServer(ctx context.Context, routerConfig *nodev1.RouterConfi
 	var graphqlPlaygroundHandler func(http.Handler) http.Handler
 
 	if r.playground {
-		r.logger.Info("Serving GraphQL playground", zap.String("url", r.baseURL))
+		r.logger.Info("Serving GraphQL playground", zap.String("url", r.baseURL+r.playgroundPath))
```

Comment:
consider using url join instead of concatenation
what happens if the baseURL doesn't have a trailing slash and the playgroundpath starts without a slash?

Reply:
playgroundPath defaults to "/", but isn't this different to the current behaviour? Before, it's just the baseURL. Now it's the baseURL + "/"

Perhaps I am misunderstanding something.

Reply:
Done

Reply:
Reading further ahead makes it clear :D

Reply:
Default was not changed.